lib/api/core/models/repositories/userRepository.js

```diff
@@ -219,7 +219,13 @@ module.exports = function (kuzzle) {
     return result;
   };
 
-  UserRepository.prototype.serializeToDatabase = UserRepository.prototype.serializeToCache;
+  UserRepository.prototype.serializeToDatabase = function (user) {
```

Comment:
Can be simplified to `UserRepository.prototype.serializeToDatabase = UserRepository.prototype.serializeToCache`